src/codegen_call_user.cpp

```
@@ -0,0 +1,664 @@
+    auto emitMockEnsureChecks = [&](llvm::Value *retVal) {
+        if (!matchedEntry || !matchedEntry->postconditions || matchedEntry->postconditions->empty() ||
+            !matchedEntry->ensureBindings)
+            return;
+
+        pushScope();
+        bindMockContractParams();
+        auto &bindings = *matchedEntry->ensureBindings;
+        if (bindings.size() == 1) {
+            bindContractValue(bindings[0], retVal, nullptr);
+        } else {
+            auto *structTy = llvm::dyn_cast<llvm::StructType>(retVal->getType());
```

Clang (14.0.6)

[warning] variable 'structTy' is not initialized

(cppcoreguidelines-init-variables)